test/tools/WebListener/README.md

```diff
@@ -30,6 +30,10 @@ $Listener = Start-WebListener -HttpPort 8083 -HttpsPort 8084
 
 # Tests
 
+## / or /Home/
+
+Returns a static HTML page containing links and descriptions of the available tests in WebListener.
```

Comment:
I think we could add that the page can be used as default for some test as we do in HTTPS tests already.

Reply:
heh. I had a blurb about that but removed it thinking it was too wordy (I have a tendency to be overly verbose). I'll add it back.